carbon-design-system/carbon
# fix(tag): add default cursor to non-filter tag

Closes #5133 

Adding the button reset caused all tag variants to get a pointer cursor. This specifys a default cursor for non-filter tags. 🏄 

#### Changelog

**New**

- adds `cursor: default;` to _tag.scss

#### Testing / Reviewing

- Check the default tag and see no cursor change on hover
- Check the filter tag and see a pointer cursor on hover
